hack/test.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+DOWN=${DOWN:true}
```

Comment:
Should this be `${DOWN:-true}`?  I am unfamiliar with just a `:`.

Reply:
yes